src/assests/styles/video-card-page.css

```diff
@@ -5,21 +5,85 @@ section {
 }
 
 .video-item {
-    width: 90%;
+    width: -webkit-fill-available;
     height: 25rem;
 }
 
-.video-card-page{
-    display:flex;
-    gap:1rem;
+.video-card-page {
+    display: flex;
+    gap: 10px;
 }
 
-.video-card-page article{
-    width:70%;
+.video-card-page article {
+    width: 80%;
     justify-content: flex-start;
 }
 
-.video-description{
+.video-description {
     text-align: justify;
-    width:90%
+    width: 90%
+}
+
+@media only screen and (max-width:1000px) {
+    #video-btn .btns .btn {
+        font-size: 1rem;
+        margin: 2px;
+        gap: 1px;
+        padding: 5px;
+    }
+
+    .card-btn {
+        padding: 0;
+    }
+
+    .must-watch-video .video .image .img {
+        width: 210px;
+        height: 180px;
+    }
+
+    .must-watch-video .video .container {
+        width: 13rem
+    }
+
+    .must-watch-video .video {
+        margin: 10px;
+        padding: 5px;
+    }
+
+}
+
+@media only screen and (max-width:800px) {
+    .video-card-page article {
+        margin: 0 1rem;
+    }
+
+    .video-card-page {
+        flex-direction: column;
+    }
+
+    .video-card-page aside .must-watch-video {
+        display: flex;
+        flex-direction: row;
```

Comment:
```row``` is by default direction

Reply:
It was not working so I added the direction as row